plugin.video.tweakers/addon.xml

```diff
@@ -15,19 +15,23 @@
     <provides>video</provides>
   </extension>
   <extension point="xbmc.addon.metadata">
-  	<platform>all</platform>
-	<summary lang="en">Watch tech videos from Tweakers.net (dutch)</summary>
-	<description lang="en">Watch tech videos from Tweakers.net (dutch)</description>
-	<disclaimer lang="en">For bugs, requests or general questions visit the tweakers.net thread on the Kodi forum.</disclaimer>
-	<summary lang="nl">Bekijk technologie videos van Tweakers.net (dutch)</summary>
-	<description lang="nl">Bekijk technologie videos van Tweakers.net (dutch)</description>
-	<disclaimer lang="nl">Bugs of andere feedback op deze plugin kan geplaatst worden in de Tweakers.net thread op het Kodi forum.</disclaimer>
+	<summary lang="en_GB">Watch tech videos from Tweakers.net (dutch)</summary>
+	    <description lang="en_GB">Watch tech videos from Tweakers.net (dutch)</description>
+	    <disclaimer lang="en_GB">For bugs, requests or general questions visit the tweakers.net thread on the Kodi forum.</disclaimer>
+	<summary lang="nl_NL">Bekijk technologie videos van Tweakers.net (dutch)</summary>
+	    <description lang="nl_NL">Bekijk technologie videos van Tweakers.net (dutch)</description>
+	    <disclaimer lang="nl_NL">Bugs of andere feedback op deze plugin kan geplaatst worden in de Tweakers.net thread op het Kodi forum.</disclaimer>
```

Comment:
is `<LANGUAGE>_<COUNTRY>` compatible with gotham?

Reply:
hmm, i would think not. I'm gonna fix it.